### ISSUE TITLE
Revise Replay UI to use CSS grid

### ISSUE THREAD

**Issue by machawk1** [oduwsdl/ipwb#600]
In #495 and #502, some issues of the replay user interface were highlighted. In each, the recommendation was to move the UI to use CSS Grid, which was the consensus. No GitHub issue previously existed to explicitly accomplish this. This issue is _that_ issue. :-)

The elements to be included in the main replay grid layout, per the above tickets, will be reduced as some elements are moved to the `/ipwbadmin` endpoint. An expectation is that users that are not running the replay system themselves (i.e., accessing others' replay systems) should not necessarily be able to upload WARCs, know where on the replay's file system indexes are stored, etc.

To be included:
* ipwb logo
* input text field for URI-R entry
* `Search URL in the archive` submit button
* NEW: admin link

This would remove from the main replay interface and move to the admin interface:
* X mementos available
* Everything under Options and Details (currently inclusive of Archive Index(es), Service Worker Version, and Upload WARC titles and corresponding values/buttons.
* IPFS WebUI and IPWB Help links

I do self-question whether the ServiceWorker buttons should be removed, since their intention is to be leveraged by the user viewing the archive (perhaps without admin permissions) and not necessarily the archive maintainer. However, the buttons relating to the SW might be a little too low-level for the casual user, e.g., does someone viewing the archive _really_ need to know the SW details? A "Fix" button or something of the like might be a better solution in lieu to be present on the main replay interface.

@ibnesayeed Please provide feedback on the above, since you have repeatedly suggested we move to use Grid.

**Comment by ibnesayeed** [comment]
We should also include cards (or a brief list) of featured resources on the landing page.

We do not need any buttons or UI to play with the SW, it perhaps was a good idea to have it there for debugging initially while we were still experimenting with it, but this should not be something users should even know about. We just need to do the right thing behind the scene.

**Comment by machawk1** [comment]
The card approach toward displaying the holdings is interesting as an alternative to a list of URIs/titles.

I think there still ought to be something on the replay interface, but much more subtle, to fix issues like cached SWs. Browsers are way too aggressive with caching of SW scripts as the state of the art to allow this behavior to be the culprit of unintended behavior. For instance, I have cleared SWs using devtools in testing yet the cleared one is still used on replay. Switching to a clean Incognito for testing helps but can't be expected for users.

**Comment by machawk1** [comment]
The list of URI-Rs and corresponding datetimes was ported to the `ipwbadmin` page in #712, so it might be time to remove that from the main landing page.

@ibnesayeed noted that this list, where ever it resides, is not scalable. Perhaps it should be fetched and parsed on-demand? How useful is such a list? What are other ways to visually present a "collection" like this?
